src/Microsoft.ML.DnnImageFeaturizer.ResNet18/ResNet18Extension.cs

```diff
@@ -25,7 +25,8 @@ public static class ResNet18Extension
         /// </summary>
         public static EstimatorChain<ColumnCopyingTransformer> ResNet18(this DnnImageModelSelector dnnModelContext, IHostEnvironment env, string outputColumnName, string inputColumnName)
         {
-            return ResNet18(dnnModelContext, env, outputColumnName, inputColumnName, Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "DnnImageModels"));
```

Comment:
We've gone through a few iterations of this code (see the git history on this file).

I don't see an issue being referenced for justifying this change. Can you add a scenario/reason why this is being changed?

Reply:
Sure
#4173